integration/grpc/pom.xml

```diff
@@ -99,4 +98,33 @@
         </plugins>
     </build>
 
+    <profiles>
+        <!-- Profile for x86 architecture -->
+        <profile>
+            <id>x86_64</id>
```

Comment:
For x86_64 and aarch64 platform users, is it possible to use the same maven command without specifying the profile, otherwise this PR will only upgrade the dependency version from before.

Reply:
It looks like there's a compatibility issue here after the upgrade. https://github.com/apache/incubator-seata/actions/runs/10665361003/job/29558555145?pr=6803